eng/mgmt/automation/swagger_to_sdk_config.json

```diff
@@ -10,7 +10,12 @@
   },
   "generateOptions": {
     "generateScript": {
-      "path": "./eng/mgmt/automation/generate.py"
+      "path": "./eng/mgmt/automation/generate.py",
+      "logPrefix": "Java-Sdk-Automation",
+      "stderr": {
+        "showInComment": "(\\[GENERATE\\]\\[Error\\]|\\[COMPILE\\] Maven build fail|ExternalError).*",
```

Comment:
When do we use `ExternalError`?

Reply:
tsp-client would log this when tsp compiler/emitter fails.

Reply:
I see.